doc/contributing/diagnostic-tooling-support-tiers.md

```diff
@@ -43,9 +43,10 @@ the following tiers.
     organization or website;
   * The tool must be open source.
 
-* Tier 3 - If possible its test suite
-  will be run at least nightly in the Node.js CI and issues opened for
-  failures.  Does not block shipping a release.
+* Tier 3 - Its test suite will be run either as nightly in Node.js CI
+  or in CITGM (in case if a test cannot be configured and run in nightly
+  by virtue of the nature of the tool in question) and issues opened for
+  failures. Does not block shipping a release.
```

Comment:
```suggestion
* Tier 3 - If possible its test suite will be run at least nightly in the
  Node.js CI or in CITGM and issues opened for failures. Does not block shipping
  a release.
```
I think we can just add the `or in CITGM` part?

Reply:
yes, thanks. I asked the same question to @mhdawson above, if he too is fine , I will go with it.

Reply:
done